scanners/android/android.go

```diff
@@ -172,26 +193,63 @@ func (scanner *Scanner) DefaultOptions() models.OptionNode {
 
 // Configs ...
 func (scanner *Scanner) Configs(repoAccess models.RepoAccess) (models.BitriseConfigMap, error) {
-	configBuilder := scanner.generateConfigBuilder(repoAccess)
+	bitriseDataMap := models.BitriseConfigMap{}
+	params := configBuildingParameters(scanner.Projects)
+	for _, param := range params {
+		configBuilder := scanner.generateConfigBuilder(repoAccess, param.useKotlinScript)
 
-	config, err := configBuilder.Generate(ScannerName)
-	if err != nil {
-		return models.BitriseConfigMap{}, err
+		config, err := configBuilder.Generate(ScannerName)
+		if err != nil {
+			return models.BitriseConfigMap{}, err
+		}
+
+		data, err := yaml.Marshal(config)
+		if err != nil {
+			return models.BitriseConfigMap{}, err
+		}
+
+		bitriseDataMap[param.name] = string(data)
 	}
 
-	data, err := yaml.Marshal(config)
-	if err != nil {
-		return models.BitriseConfigMap{}, err
+	return bitriseDataMap, nil
+}
+
+type configBuildingParams struct {
+	name            string
+	useKotlinScript bool
+}
+
+func configBuildingParameters(projects []Project) []configBuildingParams {
+	regularProjectCount := 0
+	kotlinBuildScriptProjectCount := 0
+
+	for _, project := range projects {
+		if project.UsesKotlinBuildScript {
+			kotlinBuildScriptProjectCount += 1
+		} else {
+			regularProjectCount += 1
+		}
 	}
 
-	return models.BitriseConfigMap{
-		ConfigName: string(data),
-	}, nil
+	var params []configBuildingParams
+	if 0 < regularProjectCount {
+		params = append(params, configBuildingParams{
+			name:            ConfigName,
+			useKotlinScript: false,
+		})
+	}
+	if 0 < kotlinBuildScriptProjectCount {
+		params = append(params, configBuildingParams{
+			name:            ConfigNameKotlinScript,
+			useKotlinScript: true,
+		})
+	}
+	return params
 }
 
 // DefaultConfigs ...
 func (scanner *Scanner) DefaultConfigs() (models.BitriseConfigMap, error) {
-	configBuilder := scanner.generateConfigBuilder(models.RepoAccessUnknown)
+	configBuilder := scanner.generateConfigBuilder(models.RepoAccessUnknown, false)
```

Comment:
Do we intentionally not add support for Kotlin build script-based android projects when manual config is selected?

Reply:
I did it on purpose because that means we need to ask the user an extra question about their usage of Kotlin build scripts. Right know we ask the project location, module and variant. I was thinking that I do not want to add an extra question which lengthens the ANA flow for fixing that bug. 

It is not a hard a opinion and I can be persuaded to add it to the manual config too. It is not much work because it only needs an extra option here:
https://github.com/bitrise-io/bitrise-init/blob/8ea7233cb5f2763d4f7bcaebdc4be240acd5b23e/scanners/android/android.go#L181-L192

And making sure that both Android configs are generated for as part of the default config.

What is your opinion? Would you add it to the manual config?

Reply:
I think by default we should keep feature parity in automatic and manual configs. If we don't have other decision I would add it to the manual configs, but it can go into a separate PR.

Reply:
You convinced me and added the manual config part in this PR.